### DISCUSSION TITLE
How to include Subscription System ?

### DISCUSSION THREAD

**Discussion by ghost** [withastro/roadmap#205] [Proposal]
Hi Community 👋
I would like to integrate a membership section like Ghost.cms in Astro. How to proceed ?
Thanks

**Comment by delucis** [comment]
Hi @xyz-mtl 👋 

There are quite a few different approaches that might work I imagine and different people might recommend different things.

I might recommend checking out this submission to the recent Astro Hackathon which is a site offering coding courses where users need to have a membership to access certain content:

https://hackathon-1-0-projects.vercel.app/ssr/passle-courses

I’ll be curious to hear what others suggest.

Feel free to hop into the Astro Discord server: https://astro.build/chat There’s a very active community there and a dedicated `#support-threads` channel for questions.
